charts/datahub/values.yaml

```diff
@@ -111,6 +111,71 @@ datahubUpgrade:
         cpu: 300m
         memory: 256Mi
 
+prometheus-kafka-exporter:
+  enabled: false
+  kafkaServer:
+  - prerequisites-kafka:9092  # <<release-name>>-kafka:9092
+  # Sarama logging
+  sarama:
+    logEnabled: false
+  env: []
+  # - name: <ENV_VAR_NAME>
+  #   value: <value>
+
+  # List of additional cli arguments to configure kafka-exporter
+  # for example: --log.enable-sarama, --log.level=debug, etc.
+  # all the possible args can be found here: https://github.com/danielqsj/kafka_exporter#flags
+  extraArgs: []
```

Comment:
Why do we need all these parameters written out explicitly? Please only add what you need to change for Kafka-exporter in data hub's deployment. If most things are default, then please omit them. This is adding quite a bit to an already large values.yaml file